Mathlib/Data/Finset/Basic.lean

```diff
@@ -96,6 +96,11 @@ theorem disjoint_of_subset_iff_left_eq_empty (h : s ⊆ t) :
     Disjoint s t ↔ s = ∅ := by
   rw [disjoint_iff, inf_eq_left.mpr h, bot_eq_empty]
 
+lemma pairwiseDisjoint_iff {ι : Type*} {s : Set ι} {f : ι → Finset α} :
```

Comment:
I always forget when we can just `Sort*`, but isn't indexing types one of the cases?

Reply:
Yes, but here `Set ι` forces `Type _`